src/frontend/next/src/components/BackToTopButton/BackToTopButton.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+const BackToTopButton: FC = () => <h3>BackToTopButton Component</h3>;
+
+export default BackToTopButton;
```

Comment:
In a few of our current PRs we're adding an `index.ts` file for easier imports, [see here](https://github.com/Seneca-CDOT/telescope/pull/1513/files#diff-23afde8fc396f80b6fe6e1c7acf70eebf18273d03d482bc0266ae5671cadda4cR1) for an example, can you convert `BackToTopButton`, `Banner`, and `Posts` into that? The imports will need to get changed after too 😄

Reply:
This is a great point to raise, @chrispinkney.  Here are my thoughts:

- JSX (and for TypeScript, TSX) is a non-standard extension of JavaScript
- Some people, especially outside the React ecosystem, don't like using `.js` for a file that isn't standard JavaScript (or `.ts` for TypeScript). 
- Transpilers (i.e., Babel, which translates JSX to pure JS) doesn't care what you call it, and at runtime, there is no-such thing as JSX: it all gets converted to pure functions.
- The general advice with transpiled code is to import it **without** the extension: `import Foo from './foo'` without any extension, and let the module system figure it out at build- vs. run-time.
- We are using React and next.js, so the vast majority of files are likely to contain JSX (i.e., it's not an edge case we need to worry about).

I think that another reason we used the `index.js` approach to do the final export was that we also wanted to put CSS files in the same scope:

```
/Foo
    index.js    <---- takes care of combining the JSX and CSS, and provides the default export for Foo
    Foo.jsx
    Foo.css
```

Later, we moved to Material UI, and with it, CSS-in-JS.  Next.js doesn't really like global CSS anyway (they either want you to use CSS Modules or CSS-in-JS).

In summary, I think we can ditch the `index.js` export files completely.  If we follow the patterns in the [next.js TypeScript example](https://github.com/vercel/next.js/tree/canary/examples/with-typescript), in this PR I would suggest we do this:

`src/frontend/next/src/components/BackToTopButton/BackToTopButton.tsx` becomes ` src/frontend/next/src/components/BackToTopButton.tsx` (i.e., drop the directory and use `tsx`).

Reply:
Filed https://github.com/Seneca-CDOT/telescope/issues/1553 for follow-up.

Reply:
Filed https://github.com/Seneca-CDOT/telescope/issues/1554 for follow-up.